### PR TITLE
fix: declare support for react-native 0.69

### PR DESCRIPTION
**Pull request by tido64**
## Summary

AsyncStorage should still work fine with react-native 0.69

## Test Plan

n/a

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "merge-options": "^3.0.4"
   },
   "peerDependencies": {
-    "react-native": "^0.0.0-0 || 0.60 - 0.68 || 1000.0.0"
+    "react-native": "^0.0.0-0 || 0.60 - 0.69 || 1000.0.0"
```
Comment (krizzu) [inline package.json:68]: Why not we just go ahead and support any version 60+?

Comment (tido64) [reply package.json:68]: At some point native modules will be deprecated, and this is a really good way to get people to upgrade to the latest version 😛

Comment (krizzu) [reply package.json:68]: make sense!

**Review by krizzu — APPROVED** [review]

**Pull request merged**